internal/controller/sleepinfo/jsonpatch/genericresources.go

```diff
@@ -72,19 +72,33 @@ func (g genericResource) getListOptions(namespace string, target v1alpha1.PatchT
 		Limit:     500,
 	}
 
+	includeRef := g.ResourceClient.SleepInfo.GetIncludeRef()
 	excludeRef := g.ResourceClient.SleepInfo.GetExcludeRef()
+	fieldsToInclude := getFieldToInclude(includeRef, target)
+	labelsToInclude := getLabelsToInclude(includeRef)
 	fieldsToExclude := getFieldToExclude(excludeRef, target)
 	labelsToExclude := getLabelsToExclude(excludeRef)
-	if len(fieldsToExclude) > 0 {
-		fieldSelector, err := fields.ParseSelector(strings.Join(fieldsToExclude, ","))
+
+	// Combine fields to include and exclude into a single field selector
+	var fieldSelectors []string
+	fieldSelectors = append(fieldSelectors, fieldsToInclude...)
+	fieldSelectors = append(fieldSelectors, fieldsToExclude...)
+
+	if len(fieldSelectors) > 0 {
+		fieldSelector, err := fields.ParseSelector(strings.Join(fieldSelectors, ","))
 		if err != nil {
 			return nil, err
 		}
 		listOptions.FieldSelector = fieldSelector
 	}
 
-	if len(labelsToExclude) > 0 {
-		labelSelector, err := labels.Parse(strings.Join(labelsToExclude, ","))
+	// Combine labels to include and exclude into a single label selector
+	var labelSelectors []string
+	labelSelectors = append(labelSelectors, labelsToInclude...)
+	labelSelectors = append(labelSelectors, labelsToExclude...)
```

Comment:
```suggestion
	var labelSelectors []string
	labelSelectors = append(labelSelectors, labelsToInclude..., labelsToExclude...)
```

Reply:
AFAIK, Go doesn't support using multiple variadic arguments with append. Are you sure this is valid usage? Btw, my compiler also complains about that.

Reply:
You're right! Sorry for the confusion